tx_evaluation.go

```diff
@@ -55,6 +63,14 @@ func (c *Client) EvaluateTx(ctx context.Context, data string) (response *Evaluat
 	return readEvaluateTx(raw)
 }
 
+func (c *Client) EvaluateTx(ctx context.Context, data string) (response *EvaluateTxResponse, err error) {
+	return evaluateTx(ctx, data, nil)
+}
+
+func (c *Client) EvaluateTxWithAdditionalUtxos(ctx context.Context, data string, additionalUtxos []statequery.TxOut) (response *EvaluateTxResponse, err error) {
```

Comment:
i still wish we could merge statequery.TxOut with chainsync.TxOut heh

Reply:
`statequery.TxOut` is really a "UTXO" since it has a txid and an index, whereas `chainsync.TxOut` is the same thing without those fields, so it is really a proper TxOut. the names would make much more sense if we moved `statequery.TxOut` to `shared` and renamed it to UTXO

Reply:
or TxOutRef